ingest/validation/validate_metadata.py

```diff
@@ -7,9 +7,14 @@
 participating under the convention.
 
 EXAMPLE
-# Using JSON file for Alexandria metadata convention TSV, validate input TSV
-$ python3 validate_metadata.py ../../tests/data/AMC_v1.1.3.json ../../tests/data/valid_no_array_v1.1.3.tsv \
-          --study-accession SCP123 --study-id 5dfa6718421aa90fea085476 --study-file-id 5e27451e2209c211b1e7c9cc
+# Using JSON file for latest Alexandria metadata convention in repo, validate input TSV
+$ python3 validate_metadata.py  ../../tests/data/valid_no_array_v1.2.0.tsv
```

Comment:
```suggestion
$ python3 validate_metadata.py  ../../tests/data/valid_no_array_v2.0.0.tsv
```

And similarly for other references to 1.2.0.

Reply:
`f204f50`